src/tests/Microsoft.PowerFx.Interpreter.Tests/CustomFunctions.cs

```diff
@@ -298,7 +298,7 @@ private void TestCustomFunctionWithRecordArgument(ReflectionFunction reflectionF
             var check = engine.Check(reflectionFunction.GetFunctionName() + inputRecord);
 
             FormulaValue result = null; 
-            ConfiguredTaskAwaitable<FormulaValue> resultAsync;
+            ConfiguredTaskAwaitable<FormulaValue> resultAsync = new ConfiguredTaskAwaitable<FormulaValue>();
```

Comment:
> new ConfiguredTaskAwaitable<FormulaValue>() [](http://example.com/codeflow?start=63&length=44)

Can set to null (no need to create this object that won't be used)

Reply:
cannot set this to null because ConfiguredTaskAwaitable is non-null variable type